src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/example/SecurityRatingExample.java

```diff
@@ -44,8 +42,8 @@ public void makeImmutable() {
    * Initializes a security rating with SecurityScoreExample.
    */
   @JsonCreator
```

Comment:
Is this still required, if there is no `@JsonProperty`?

Reply:
Right. I've removed it.